translations/strings.yaml

```diff
@@ -266,7 +266,7 @@ systray:
   quit: Quit Mozilla VPN
 
 settings:
-  preferences: Preferences
-
+  systemPreferences: System preferences
```

Comment:
Is there a reason to go with "Preferences" and not "Settings"?

Reply:
I'm not sure but @chrisrooneys or @jess-cook03 may have more context on this choice.